bftengine/src/simplestatetransfer/SimpleStateTran.cpp

```diff
@@ -279,7 +278,7 @@ SimpleStateTran::SimpleStateTran(
   config.cVal = cVal;
   config.numReplicas = 3 * fVal + 2 * cVal + 1;
   config.pedanticChecks = pedanticChecks;
-  auto comparator = concord::storage::memorydb::KeyComparator(new concord::storage::blockchain::DBKeyComparator());
+  auto comparator = concord::storage::memorydb::KeyComparator();
```

Comment:
If I am not mistaken, removing the comparator will force lexicographical comparison. Will that be OK in this context?